### ISSUE TITLE
Icon theme doesn't appear in Linux Mint Cinnamon

### ISSUE THREAD

**Issue by Growebis** [vinceliuice/McMojave-circle#68]
Using Linux Mint Cinnamon 21.3, it doesnt appear at all in the themes setting. as demonstrated in the picture below.
![image](https://github.com/vinceliuice/McMojave-circle/assets/113867283/d55edd2f-92b1-44d9-a435-6490ad5df925)
Even though i have it both in my .icons folder and /usr/share/icons
![image](https://github.com/vinceliuice/McMojave-circle/assets/113867283/b563f3b5-7a3c-44b1-a190-9135cb031e1f)
Any help would be appreciated.

**Comment by haggen88** [comment]
@Growebis  do you have them installed on /home/USER/.local/share/icons  ?

**Comment by Growebis** [comment]
Yeah i do, anyways i figured out the problem and the fix was pretty simple, after extracting the icon pack i didn't know that there the different types of mcmojave were in it so i just pulled them out and bam, it's fixed.
